Wikipedia/Localizations/qqq.lproj/Localizable.strings

```diff
@@ -136,7 +136,7 @@
 "wikitext-upload-save-anonymously" = "Button text for publishing without logging in";
 "wikitext-upload-save-anonymously-warning" = "Button sub-text informing user or draw-backs of not signing in before saving wikitext. Parameters:\n* $1 - sign in button text";
 "wikitext-upload-save-terms-and-license" = "Button text for information about the Terms of Use and edit license. Parameters:\n* $1 - 'Terms of Use' link ([[Wikimedia:Wikipedia-ios-wikitext-upload-save-terms-name]])\n* $2 - license name link";
-"wikitext-upload-save-terms-name" = "This message is used as $1 in the message [[Wikimedia:Wikipedia-ios-wikitext-upload-save-terms-and-license]].\n{{Identical|Terms of use}}";
+"wikitext-upload-save-terms-name" = "This message is used in the message [[Wikimedia:Wikipedia-ios-wikitext-upload-save-terms-and-license]].\n{{Identical|Terms of use}}";
```

Comment:
@montehurd This one was the only one I struggled with. The `$1` is confusing my very naive algorithm since it refers to a substation is a another string, not this one. I took it out so the test would pass. Thoughts?